packages/apps/src/overlays/Attest.tsx

```diff
@@ -18,13 +19,28 @@ interface Props {
 function Attest ({ className }: Props): React.ReactElement<Props> | null {
   const { t } = useTranslation();
   const { allAccounts } = useAccounts();
-  const { api } = useApi();
+  const { api, isApiReady } = useApi();
+  const [isOldClaimProcess, setIsOldClaimProcess] = useState(true);
+  const [needAttest, setNeedAttest] = useState<Option<EthereumAddress>[]>([]);
+
+  useEffect(() => {
+    if (isApiReady && typeof api.query?.claims?.preclaims !== 'undefined') {
+      setIsOldClaimProcess(false);
+    }
+  }, [api, isApiReady]);
```

Comment:
VSCode would like to add `api.query.claims.preclaims` to the depandancy array, but this makes `apps` to crash because it's not available when being loaded.